includes/admin/tools/import/import-actions.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Exports Give's core settings.
+ *
+ * Give_Core_Settings class.
+ *
+ * @since 1.5
+ * @return void
+ */
+function give_core_settings_import() {
+
+	$json_string   = file_get_contents( $_FILES['json_file']['tmp_name'] );
+	$json_to_array = json_decode( $json_string, true );
+
+	// Unset General Pages.
+	unset( $json_to_array['success_page'], $json_to_array['failure_page'], $json_to_array['history_page'] );
+
+	// Featured image sizes import.
+	if ( 'enabled' === $json_to_array['form_featured_img'] ) {
+		$images_sizes = get_intermediate_image_sizes();
+
+		if ( ! in_array( $json_to_array['featured_image_size'], $images_sizes ) ) {
+			unset( $json_to_array['featured_image_size'] );
+		}
+	}
+
+	// Emails > Email Settings > Logo.
+	if ( array_key_exists( 'email_logo', $json_to_array ) && ! empty( $json_to_array['email_logo'] ) ) {
+
+		// Need to require these files.
+		if ( ! function_exists( 'media_handle_upload' ) ) {
+			require_once( ABSPATH . 'wp-admin/includes/image.php' );
+			require_once( ABSPATH . 'wp-admin/includes/file.php' );
+			require_once( ABSPATH . 'wp-admin/includes/media.php' );
+		}
+
+		$url = $json_to_array['email_logo'];
+		$tmp = download_url( $url );
+
+		if ( is_wp_error( $tmp ) ) {
```

Comment:
@Sidsector9 This is unnecessary code. Can you remove it?

Reply:
@ravinderk  I left it blank for any action to be taken in case of error.